src/docs/previews/accordion/colors/app.svelte

```diff
@@ -1,59 +1,64 @@
 <script lang="ts">
-    import { Accordion } from 'ato-ui';
+	import { Accordion } from 'ato-ui';
 
-    type ExampleItem = {
-        key: string;
-        title: string;
-        content: string;
-        icon: string;
-        titleBgStyle: string;
-    }
+	type ExampleItem = {
+		key: string;
+		title: string;
+		content: string;
+		icon: string;
+		titleBgStyle: string;
+	};
```

Comment:
What changes were made here? Just spacing right?

Reply:
vscode is picking up the prettier settings and I have format on save activated

Reply:
I had tried something some classes and prettier formated the file ... so if you like I can revert it

Reply:
I had changed "justify-left" to "justify-start" in that file

Line 54:
				<div class="flex justify-start items-center gap-2">

Reply:
@delight yeah no worries, looks good :)